webapp/src/components/__snapshots__/Help.test.js.snap

```diff
@@ -31,7 +31,7 @@ exports[`should take a snapshot 1`] = `
     <img
       alt="RSU Statuses Shown on Map"
       id="helpimage"
-      src="rsu_status.PNG"
+      src="[object Object]"
```

Comment:
(This doesn't need to be changed necessarily I am just curious)

Reply:
This was because in the original PR, I was using "react-scripts test" as the test runner, but in this PR, I switched to using jest. This required a fair amount of configuration, and one of those steps was to replace all non-JS files with blank objects, so that the tests could run successfully. This was kind of  a dumb solution. I just switched back to using "react-scripts test", and it is back to being image names now. I like this more than the blank objects

Reply:
Sounds good!